rubicon_ml/viz/dataframe_plot.py

```diff
@@ -89,8 +89,15 @@ def load_experiment_data(self):
         `dataframe_name` must have the same schema.
         """
         self.data_df = None
+        count = 0
 
         for experiment in self.experiments:
+            if len(experiment.dataframes()) == 0:
+                print(
+                    f"WARNING: Experiment {experiment.name} does not have any dataframes logged to it. "
+                )
+                continue
+            count += 1
             dataframe = experiment.dataframe(name=self.dataframe_name)
```

Comment:
we also want warn if there is a dataframe, but it isn't named `dataframe_name`. right now, we'd still get an error here if there is no dataframe named `dataframe_name`

instead of checking if `len(experiment.dataframes()) == 0` and warning, we should wrap this getter in a try/except and warn instead of erroring in the except block